charon/src/bin/charon/main.rs

```diff
@@ -47,6 +47,8 @@ use charon_lib::logger;
 use charon_lib::options;
 use charon_lib::trace;
 
+/// Rename this module once subcommand migration finishes.
+mod cli_rework;
```

Comment:
Can you move the contents of `cli_rework` to the existing `charon_lib::options` module?